README.md

```diff
@@ -15,4 +16,21 @@ npm init @eslint/config
 ```bash
 # use npx
 npx @eslint/create-config
-```
\ No newline at end of file
+```
+
+If you want to use the specified shared configs:
+
+```bash
+# use `eslint-config-eslint` shared config
+npm init @eslint/config --config eslint
+# or (`eslint-config` prefix is optional)
+npm init @eslint/config --config eslint-config-eslint
```

Comment:
```suggestion
# use `eslint-config-semistandard` shared config
npm init @eslint/config --config semistandard
# or (`eslint-config` prefix is optional)
npm init @eslint/config --config eslint-config-semistandard
```

I think `eslint-config-eslint` could be confusing. It might be better to use something like `semistandard` for the example?